website/static/css/pages/profile-page.css

```diff
@@ -4,22 +4,19 @@
 }
 
 span.edit-profile-settings {
-    display: block;
-    position: absolute;
-    bottom: 0;
+    float:right;
+    top: 0;
     right: 0;
 }
 
 div.profile-fullname{
     display: block;
-    position: relative;
     bottom: 0;
 }
 
 span#profileFullname{
-    position: absolute;
-    bottom: 0;
-    margin-bottom: 20px;
+    display:inline;
+    margin-left: -2px;
```

Comment:
It's not ideal to correct margins with - sizes. Check what other css is effecting margin here, possibly remove m-l-sm. 
See style guide for margin and padding helpers https://centerforopenscience.github.io/osf-style/style.html#layout-helper